### ISSUE TITLE
ESP 32

### ISSUE THREAD

**Issue by Nichelsar** [AronRubin/SeeedHeartBreathRadar#2]
Hi, not really a problem, but have you been able to use an esp32 to receive the data?

I'm looking for a sensor that monitors the heart rate of a human, that can be placed as far away as possible, and this seems, the maximum is 2 meters.
Have you tried it at a higher distance?

Thanks for your work, you are a crack

**Comment by AronRubin** [comment]
The project that I created this library for has been on the back burner but eventually it will target an esp32. The initial development used a rp2040 based board. It is not obvious to me that there would be any difference but esp32 is a little funny under the hood. Are you experiencing any specific issues?

**Comment by Nichelsar** [comment]
No, at the moment I am looking at documentation to see if I finally buy the sensor.
I don't have any specific problem.
I'm very green on esp32 and I needed to know if you had something done.
Don't worry, I'm still investigating, thanks for your attention

**Comment by AronRubin** [comment]
Also 2 meters does seem to be a hard limit at the power level of the emitter at 60GHz. Infineon might allow a higher level but that could impact the safety assessment (and power consumption).

**Comment by Nichelsar** [comment]
oh ok.
In my project power consumption is not a problem, but I take it into account.
Thank you very much!! you are very good person

**Comment by DuncanIdahoCT** [comment]
On the subject of ESP32:
I’m using the Seeed example Arduino IDE sketch for the Fall Detection module with an ESP32 just fine. You need to add support for ESP32 by looking for “ESP32 by Espressif Systems” in the Arduino IDE boards manager.
Note:
I found that on ESP3 MCU’s (most/all?) that you simply need to change to Serial2 as the main serial is obviously the USB com port and Serial1 is otherwise reserved… SoftSerial is not needed as is often suggested with ESP32 vs. Arduino. I used pins 23,5 for RX,TX in my tests.

**Comment by Nichelsar** [comment]
Hello, thanks for commenting. I've gotten the heartbeat detector to work. But the breath sensor doesn't. Could you pass the code you use? Thanks

**Comment by DuncanIdahoCT** [comment]
Hi, I don’t actually have one of the Heart/Breathe sensor modules, I’m using this code on a Fall Detection version so I can’t confirm what works or not for you. I may have one of the modules you have in a week perhaps, I ordered one for testing as I think it’s more useful than the fall detection.

**Comment by Nichelsar** [comment]
Ahhh ok, good loock
